packages/cli/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@e2b/cli",
-  "version": "1.0.8",
+  "version": "1.0.9",
```

This will be increased automatically
```suggestion
  "version": "1.0.8",
```